anndata/tests/test_backed_sparse.py

```diff
@@ -149,26 +156,40 @@ def make_one_elem_mask(size: int) -> np.ndarray:
 
 # test behavior from https://github.com/scverse/anndata/pull/1233
 @pytest.mark.parametrize(
-    "make_bool_mask",
+    "make_bool_mask,should_trigger_optimization",
     [
-        make_randomized_mask,
-        make_alternating_mask,
-        make_one_group_mask,
-        make_one_elem_mask,
+        (make_randomized_mask, None),
+        (make_alternating_mask_10, True),
+        (make_alternating_mask_5, False),
+        (make_one_group_mask, True),
+        (make_one_elem_mask, False),
     ],
-    ids=["randomized", "alternating", "one_group", "one_elem"],
+    ids=["randomized", "alternating_10", "alternating_5", "one_group", "one_elem"],
 )
 def test_consecutive_bool(
+    mocker,
     ondisk_equivalent_adata: tuple[AnnData, AnnData, AnnData, AnnData],
     make_bool_mask: Callable[[int], np.ndarray],
+    should_trigger_optimization: bool | None,
```

Comment:
Could you document this value? I think saying what the expected optimization is should be fine.